scripts/qa_crowbarsetup.sh

```diff
@@ -2262,7 +2262,7 @@ function enable_ssl_generic()
             if [[ ! $(iscloudver 7plus) ]] ; then
                 return
             fi
-        ;;&
```

Comment:
here is error from jenkins job 
14:18:42 qa_crowbarsetup.sh: line 2265: syntax error near unexpected token `&'
14:18:42 qa_crowbarsetup.sh: line 2265: `        ;;&'

https://ci.suse.de/job/openstack-mkcloud/22492/console